File: MinMaxSum_Array.py
def minMaxsum(numbers_array):
  #Sorting the array
    numbers_array.sort()
  #Finding the length of array
    total_length=len(numbers_array)
  #Setting values of min_sum and max_sum to 0 as default
    min_sum=0
    max_sum=0
  #One less than total length of array, i.e the Second last position of array.
    second_last=total_length-1
    for n in range(total_length-1):
        min_sum+=numbers_array[n]
        max_sum+=numbers_array[second_last]
        second_last-=1
    print(min_sum,max_sum)
        
minMaxsum([1,2,3,4])

File: MinMaxSum_Array.js
function Min_MaxSum(arr){
  //Length of array
    let arr_length=arr.length;
  //One less than length of array i.e second last position of array
    let second_last=arr_length-1;
  // Setting the min_sum and max_sum values to 0
    let min_sum=0;
    let max_sum=0;
    for(let i=0;i<second_last;i++){
        min_sum+=arr[i];
        max_sum+=arr[second_last];
        second_last-=1;
    }
    console.log(min_sum,max_sum);
}
var arr=[2,4,6,8];
Min_MaxSum(arr);


